packages/core/stories/patterns/forms/forms.stories.tsx

```diff
@@ -0,0 +1,324 @@
+  const [value, setValue] = useState<string>("Value text");
+  const [isError, setIsError] = useState<boolean>(false);
+  const MAX_CHARS = 1000;
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const newVal = event.target.value;
+    setValue(newVal);
+    setIsError(newVal.length > MAX_CHARS);
+  };
+
+  return (
+    <GridLayout
+      columns={4}
+      style={{ width: "calc(var(--salt-size-base) * 12)" }}
+    >
+      <GridItem colSpan={4}>
+        <FormField>
+          <FormFieldLabel>Field label</FormFieldLabel>
+          <Input defaultValue="Value text" />
+          <FormFieldHelperText>Helper text</FormFieldHelperText>
+        </FormField>
+      </GridItem>
+      <GridItem colSpan={2}>
+        <FormField>
+          <FormFieldLabel>Field label</FormFieldLabel>
+          <Dropdown defaultSelected={["Value text"]} style={{ width: "100%" }}>
+            <Option value="Value text">Value text</Option>
+          </Dropdown>
+          <FormFieldHelperText>Helper text</FormFieldHelperText>
+        </FormField>
+      </GridItem>
+      <GridItem colSpan={3}>
+        <FormField>
+          <FormFieldLabel>Field label</FormFieldLabel>
+          <Dropdown defaultSelected={["Value text"]} style={{ width: "100%" }}>
+            <Option value="Value text">Value text</Option>
+          </Dropdown>
+          <FormFieldHelperText>Helper text</FormFieldHelperText>
+        </FormField>
+      </GridItem>
+      <GridItem colSpan={4}>
+        <FormField>
+          <FormFieldLabel>Field label</FormFieldLabel>
+          <MultilineInput
+            bordered
+            endAdornment={
+              <Label variant={!isError ? "secondary" : "primary"}>
+                {!isError && `${value.length}/${MAX_CHARS}`}
+                {isError && <strong>{`${value.length}/${MAX_CHARS}`}</strong>}
+              </Label>
+            }
+            onChange={handleChange}
+            value={value}
+            validationStatus={isError ? "error" : undefined}
+          />
+          <FormFieldHelperText>Helper text</FormFieldHelperText>
+        </FormField>
+      </GridItem>
+    </GridLayout>
+  );
+};
+
+export const Sections = () => {
+  return (
+    <GridLayout
+      columns={4}
+      style={{ width: "calc(var(--salt-size-base) * 20)" }}
+    >
+      <GridItem colSpan={2}>
+        <FormField>
+          <FormFieldLabel>Expected total annual volumes</FormFieldLabel>
+          <Input placeholder="e.g., 100000" />
+        </FormField>
+      </GridItem>
+      <GridItem colSpan={2} />
+      <GridItem colSpan={2}>
+        <FormField>
+          <FormFieldLabel>Expected total annual valurs</FormFieldLabel>
+          <StackLayout direction="row" align="center">
+            <Input placeholder="e.g., 100000" />
+            <Text style={{ whiteSpace: "nowrap" }}>US Dollar</Text>
+          </StackLayout>
+        </FormField>
+      </GridItem>
+      <GridItem colSpan={4}>
+        <hr></hr>
```

Comment:
Different color to design? Worth thinking implement a component with primary/secondary/tertiary separable token.

Reply:
Will have proper design done in #3320

Reply:
changed to use a border, can be updated after design